R-package/R/lgb.cv.R

```diff
@@ -295,7 +295,9 @@ lgb.cv <- function(params = list()
 
   # Cannot use early stopping with 'dart' boosting
   if (using_dart) {
-    warning("Early stopping is not available in 'dart' mode.")
+    if (using_early_stopping)
+      warning("Early stopping is not available in 'dart' mode.")
+
```

Comment:
```suggestion
    if (using_early_stopping){
      warning("Early stopping is not available in 'dart' mode.")
    }
```

Please use braces here. Please try running the linter to see other style recommendations for this repo.

```shell
Rscript .ci/lint-r-code.R R-package/
```